functional_tests/steps/statistical_article_page.py

```diff
@@ -568,18 +568,22 @@ def user_selects_featured_chart_preview_mode(context: Context):
     context.page.wait_for_timeout(500)
     preview_mode_select = context.page.locator("#id_preview_mode")
     preview_mode_select.select_option(value="featured_article")
+    context.page.wait_for_timeout(250)
+
+    browser_context = context.playwright_context
+    with browser_context.expect_page() as preview_page:
+        context.page.get_by_role("link", name="Preview in new tab").click()
```

Comment:
could use a comment as to why we're doing this, orther than that LGTM

Reply:
Added here [ae7dfa8](https://github.com/ONSdigital/dis-wagtail/pull/341/commits/ae7dfa861bbb5492936c0f606b491822d5c7841c)